### ISSUE TITLE
Go ahead and experiment with migration to SQLite or similar

### ISSUE THREAD

**Issue by KnowledgeGarden** [KnowledgeGarden/lite-net-3#17]
Right now, the platform, as a prototype, will not scale; it's running on NeDB, a simple local version of Mongo; it cannot do full text search even though the instructions suggest that's possible.
SQLite is used in a number of similar projects such as Zotero, and appears easy to drop in there.
That requires a bootstrap function which maps the old NeDB JSON files to SQLite tables.

In a sense, this is a prelude to other backsides for future work.

**Comment by KnowledgeGarden** [comment]
https://www.sqlitetutorial.net/sqlite-nodejs/

**Comment by KnowledgeGarden** [comment]
https://www.sqlite.org/lang.html
no joins.
Wonder what other sql db besides e.g. postgres is around.
Do we need joins?

**Comment by KnowledgeGarden** [comment]
https://www.sqlitetutorial.net/sqlite-full-text-search/

**Comment by KnowledgeGarden** [comment]
https://www.npmjs.com/package/sqlite3
and see also
https://coolaj86.com/articles/building-sqlcipher-for-node-js-on-raspberry-pi-2/

**Comment by KnowledgeGarden** [comment]
Started on this one today.

**Comment by KnowledgeGarden** [comment]
Have a driver shelled in. It compiles and runs.
This task must be completed before being able to easily delete a journal entry, which can require a lot of surgery if it has wikilinks.
